### PR TITLE
fix for  if() clauses for empty array

### PR DESCRIPTION
**Pull request by ThomasMic**
keys exists in object but we also must test values (if array is empty, enter the first if() block)

### CODE REVIEW

File: thesesfr-organisme/README.md [hunk 1]
```diff
@@ -81,32 +88,40 @@ You can use thesesfr-organisme for an enrichment process. You just add the middl
...
-You can use thesesfr-organisme for an enrichment process with [ezp](https://github.com/ezpaarse-project/node-ezpaarse) like this:
+You can use thesesfr for an enrichment process with [ezp](https://github.com/ezpaarse-project/node-ezpaarse) like this:
 
 ```bash
 # enrich with one file
 ezp process <path of your file> \
   --host <host of your ezPAARSE instance> \
   --settings <settings-id> \
-  --header "ezPAARSE-Middlewares: thesesfr-organisme" 
+  --header "ezPAARSE-Filter-Redirects: false" \
```
Comment (felixleo22) [inline thesesfr-organisme/README.md:98]: Les headers dans cette proposition ont l'air d'être calqué sur votre config et non par défaut, il voudrait mieux la retirer. (idem pour les autres middlewares)

**Review by felixleo22 — APPROVED** [review]
